dask/dataframe/tests/test_dataframe.py

```diff
@@ -402,6 +402,8 @@ def test_describe_numeric(method, test_values):
 
     test_quantiles = [0.25, 0.75]
 
+    ddf.describe(percentiles_method=method)
```

Comment:
This is back?

Reply:
Hmm - Not sure where this came from. Guess I can remove it :)